goosebit/api/v1/software/routes.py

```diff
@@ -22,7 +24,7 @@
     dependencies=[Security(validate_user_permissions, scopes=["software.read"])],
 )
 async def software_get(_: Request) -> SoftwareResponse:
-    return await SoftwareResponse.convert(await Software.all().prefetch_related("compatibility"))
+    return SoftwareResponse(software=await Software.all().prefetch_related("compatibility"))
```

Comment:
Just cosmetics: in the rest of the code this is two lines first assigning using a variable software.